src/info/manifest.c

```diff
@@ -383,6 +386,750 @@ manifestNewInternal(void)
+            strPtr(strSub(manifestName, sizeof(MANIFEST_TARGET_PGDATA))));
+
+        FUNCTION_TEST_RETURN_VOID();
+        return;
+    }
+
+    // Process storage types
+    switch (info->type)
+    {
+        // Add paths
+        // -------------------------------------------------------------------------------------------------------------------------
+        case storageTypePath:
+        {
+            // There should not be any paths in pg_tblspc
+            if (strEqZ(buildData.manifestParentName, MANIFEST_TARGET_PGDATA "/" MANIFEST_TARGET_PGTBLSPC))
+            {
+                THROW_FMT(
+                    LinkExpectedError, "'%s' is not a symlink - " MANIFEST_TARGET_PGTBLSPC " should contain only symlinks",
+                    strPtr(manifestName));
+            }
+
+            // Add path to manifest
+            ManifestPath path =
+            {
+                .name = manifestName,
+                .mode = info->mode,
+                .user = info->user,
+                .group = info->group,
+            };
+
+            manifestPathAdd(buildData.manifest, &path);
+
+            // Skip excluded path content
+            if (buildData.excludeContent != NULL && strLstExists(buildData.excludeContent, manifestName))
+            {
+                LOG_INFO(
+                    "exclude contents of '%s/%s' from backup using '%s/' exclusion", strPtr(buildData.pgPath), strPtr(info->name),
+                    strPtr(strSub(manifestName, sizeof(MANIFEST_TARGET_PGDATA))));
+
+                FUNCTION_TEST_RETURN_VOID();
+                return;
+            }
+
+            // Skip the contents of these paths if they exist in the base path since they won't be reused after recovery
+            if (strEq(buildData.manifestParentName, MANIFEST_TARGET_PGDATA_STR))
+            {
+                if (strEqZ(info->name, PG_PATH_PGDYNSHMEM) && pgVersion >= PG_VERSION_94)
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGNOTIFY))
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGREPLSLOT) && pgVersion >= PG_VERSION_94)
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGSERIAL) && pgVersion >= PG_VERSION_91)
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGSNAPSHOTS) && pgVersion >= PG_VERSION_92)
+                {
+                    FUNCTION_TEST_RETURN_VOID();
+                    return;
+                }
+
+                if (strEqZ(info->name, PG_PATH_PGSTATTMP))
```

Comment:
Have we stopped support for 8.3? In Perl this had:

Skip temporary statistics in pg_stat_tmp even when stats_temp_directory is set because PGSS_TEXT_FILE is always created
        # there.
        next if $strFile =~ ('^' . MANIFEST_PATH_PGSTATTMP . '\/') && $self->dbVersion() >= PG_VERSION_84;

Reply:
No, it just looks like this one was missed as well.  Same as above.

Reply:
Code has been changed to (commit 1fe22c3bb51dbb7070d22d8329ee999290ea8560):
`if (strEqZ(info->name, PG_PATH_PGSTATTMP) && pgVersion >= PG_VERSION_84)`